mlflow/store/sqlalchemy_store.py

```diff
@@ -55,15 +55,15 @@ def __init__(self, db_uri, default_artifact_root):
 
         :param db_uri: SQL connection string used by SQLAlchemy Engine to connect to the database.
                        Argument is expected to be in the format:
-                       ``db_type://<user_name>:<password>@<host>:<port>/<database_name>`
+                       ``db_type+driver://<user_name>:<password>@<host>:<port>/<database_name>`
                        Supported database types are ``mysql``, ``mssql``, ``sqlite``,
-                       and ``postgresql``.
+                       and ``postgresql``. The driver element is optional.
```

Comment:
Can we give some context for new users. Perhaps add a link to this [SQLAlchemy Engine Configuration](https://docs.sqlalchemy.org/en/13/core/engines.html#database-urls) page you pointed to in issue #1266. The page has a lot of specific examples as well, which is great!